.github/workflows/publish.yml

```diff
@@ -1,9 +1,16 @@
 name: Publish to pub.dev
 
 on:
-  push:
-    tags:
-      - 'v[0-9]+.[0-9]+.[0-9]+*'
```

Comment:
Why disable publishing? Do we want to create releases that are not published?

Reply:
This means to disable auto trigger on a commit with a version tag. I replace it with the manually trigger to enable dry run.